travis.yml

```diff
@@ -0,0 +1,15 @@
+language: ruby
+rvm:
+- 2.5.1
+services:
+- postgresql
+before_script:
+- bin/rails db:create
+- bin/rails db:migrate
+deploy:
+  provider: heroku
+  api_key:
+    secure: TODO
+  app: videostoreconsumerapi
```

Comment:
You can run `travis setup heroku --force` and follow the prompts to set this up.